README.md

```diff
@@ -85,13 +85,28 @@ depmod
 modprobe gve
 ```
 
+Check via `ethtool -i devname` that the new driver is installed. If not, you can
+install it manually.
+
+> [!WARNING]
+> Run this as a single line, as running `rmmod` alone will remove the existing
+> driver and disconnect you if connected over SSH.
+
+```bash
+sudo rmmod gve; sudo insmod ./build/gve.ko
+```
+
 # Configuration
 
 ## Ethtool
 
 gve supports common ethtool operations including controlling offloads and
 multi-queue operation.
 
+> [!NOTE]
+> The number of RX and TX queues must be no more than half their maximum values.
```

Comment:
This is only required when using XDP. For normal usage, we can support the maximum queue count

Reply:
Done. Moved this to an XDP subsection.